components/log-viewer-webui/server/.env

```diff
@@ -6,3 +6,5 @@ HOST=localhost
 PORT=3000
 CLP_DB_USER=clp-user
 CLP_DB_PASS=
+
+FASTIFY_CLOSE_GRACE_DELAY=1000
```

Comment:
can we configure this in the config json file since this is not credential related?